### PR TITLE
<3 Add cuwsed fuwwy twansfowmations ;-;

### PR DESCRIPTION
**Pull request by zuzak**

### CODE REVIEW

**Comment by SamStudio8** [comment]
owo what's this?

File: modules/news.js [hunk 1]
```diff
@@ -127,6 +128,9 @@ module.exports = {
           }
           // news transform
           try {
+            if (bot.config.get('news.owo')) {
```
Comment (georgewatson) [inline modules/news.js]: Would it be better to do this after the word substitutions?

**Comment by SamStudio8** [comment]
-1
modules should perhaps offer a `registerBeforeResponse` method for other modules to hook into; it should be up to the owo module to hook itself into the news feed, not the other way around

**Comment by zuzak** [comment]
[Put that idea on the backlog](https://github.com/zuzakistan/civilservant/issues/new); it's probably a sensible idea but it's a fairly major piece of work.

I don't think it's something that blocks this pull request from being merged.

**Comment by kragniz** [comment]
ok

**Pull request merged**